docs/CONTRIBUTING.md

```diff
@@ -40,6 +40,26 @@ An easier solution would be to use Docker.
 1. Run `npm test`
 1. IF eslint detect some issues run `npm run eslint-fix` before manually fixing the issue (Will save you time :smile:) and then run `npm test` again.
 
+### Common Errors
+**Linux**
+
+If you encounter this error when running redis-server:
+```
+# Could not create server TCP listening socket *:6379: bind: Address already in use
+```
+1. Run `sudo systemctl status redis` to check if the service is running
```

Comment:
This is unnecessary. Regular users should not have to access ```systemctl``` in this scenario. We are creating a docker container so errors addressed in this section should pertain to those that could be had from running redis in a docker container. Please read #135 for more information.